client/state/ui/editor/image-editor/selectors.js

```diff
@@ -74,3 +74,15 @@ export function getImageEditorCrop( state ) {
 export function getImageEditorAspectRatio( state ) {
 	return state.ui.editor.imageEditor.aspectRatio;
 }
+
+/**
+ * Returns whether the image meets the minimum dimensions for full editor capabilities
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean} whether the image meets the minimum dimensions or not
+ *
+ */
+export function getImageMeetsMinimumDimensions( state ) {
```

Comment:
A note for later that new selectors should go into the global `client/state/selectors` folder. This allows us to do more quickly find selectors, and search for them in https://wpcalypso.wordpress.com/devdocs/selectors